x-pack/test/functional/apps/infra/hosts_view.ts

```diff
@@ -150,6 +154,67 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       });
     });
 
+    describe('#Single host Flyout', () => {
+      before(async () => {
+        await Promise.all([
+          await setHostViewEnabled(true),
+          await loginWithReadOnlyUser(),
+          await pageObjects.common.navigateToApp(HOSTS_VIEW_PATH),
+          await pageObjects.timePicker.setAbsoluteRange(
+            START_HOST_PROCESSES_DATE.format(timepickerFormat),
+            END_HOST_PROCESSES_DATE.format(timepickerFormat)
+          ),
+          pageObjects.infraHostsView.clickTableOpenFlyoutButton(),
+        ]);
+      });
+
+      after(async () => {
+        await pageObjects.infraHostsView.clickCloseFlyoutButton();
+        await logoutAndDeleteReadOnlyUser();
+      });
+
+      it('should render metadata tab', async () => {
+        const metadataTab = await pageObjects.infraHostsView.getMetadataTabName();
+        expect(metadataTab).to.contain('Metadata');
+      });
+
+      describe('should render processes tab', async () => {
+        const processTitles = [
+          'Total processes',
+          'Running',
+          'Sleeping',
+          'Dead',
+          'Stopped',
+          'Idle',
+          'Zombie',
+          'Unknown',
+        ];
+
+        processTitles.forEach((value, index) => {
+          it(`Render title: ${value}`, async () => {
+            await pageObjects.infraHostsView.clickProcessesFlyoutTab();
+            const processesTitleValue =
+              await pageObjects.infraHostsView.getProcessesTabContentTitle(index);
+            const processValue = await processesTitleValue.getVisibleText();
+            expect(processValue).to.eql(value);
+          });
+        });
+
+        it('should render processes total value', async () => {
+          await pageObjects.infraHostsView.clickProcessesFlyoutTab();
+          const processesTotalValue =
+            await pageObjects.infraHostsView.getProcessesTabContentTotalValue();
+          const processValue = await processesTotalValue.getVisibleText();
+          expect(processValue).to.eql(313);
+        });
+
+        it('should render processes table', async () => {
```

Comment:
Would it be interesting to validate the content of a row? Just to make sure it's rendering some content?

Reply:
I added a check for the table body and the expand button click (if we don't have the data the table body won't be present - there is a message instead)